### PR TITLE
google analytics

### PR DESCRIPTION
**Pull request by bendecoste**

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -102,6 +102,20 @@ class Index extends React.Component {
                     <link rel='icon' href='static/favicon.ico'/>
                     <meta name='og:title' content='TF Encrypted | Encrypted Deep Learning in Tensorflow'/>
                     <meta name='og:image' content='/img/tfe-encryption.png'/>
+
+                    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-125728747-3"></script>
+                    <script dangerouslySetInnerHTML={{ __html: `
+                      if (window.location.hostname === 'localhost') {
```
Comment (ianlivingstone) [inline pages/index.js]: What should traffic on netlify?

Comment (bendecoste) [reply pages/index.js]: ?

Comment (bendecoste) [reply pages/index.js]: Turn it off on that domain as well?

Comment (ianlivingstone) [reply pages/index.js]: Yeah - so we don't muddy the results

Comment (bendecoste) [reply pages/index.js]: updated

**Pull request merged**